internal/pkg/rpaas/manager.go

```diff
@@ -154,7 +154,7 @@ type RpaasManager interface {
 	GetPlans(ctx context.Context) ([]v1alpha1.RpaasPlan, error)
 	GetFlavors(ctx context.Context) ([]Flavor, error)
 	BindApp(ctx context.Context, instanceName string, args BindAppArgs) error
-	UnbindApp(ctx context.Context, instanceName string) error
+	UnbindApp(ctx context.Context, appName, instanceName string) error
```

Comment:
The instance name should be the second parameter according to other methods around it.